bin/pycbc_banksim

```diff
@@ -374,7 +374,7 @@ with ctx:
                                       template_sample_rate,
                                       filter_N, options.filter_sample_rate)
 
-            h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
+                h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
```

Comment:
Yeah, `f_lower` is constant within the loop, but `signal_high_freq_cutoff` is signal-dependent (when using `--filter-high-freq-cutoff-isco-factor`). I should have commented on the next line, sorry!